client/src/styles/setup/_variables.scss

```diff
@@ -7,7 +7,7 @@ $chat-first-part-msg-bgd: #ffffff;
 $chat-second-part-msg-bgd: #9360ff;
 $chat-send-btn-bgd: #e5dbff;
 $scrollbar-bgd: #e5dbff;
-$dark-purple: #baa6f1; // todo: remove
+$dark-purple: #baa6f1;
```

Comment:
give meaningful color names
$primary-color: var(--red)
such that if u change the color u will change it once as u use the name primary-color on the app